var/spack/repos/builtin/packages/heffte/package.py

```diff
@@ -128,12 +128,18 @@ def cmake_args(self):
 
     @run_after("install")
     def setup_smoke_test(self):
+        if self.spec.satisfies("@:2.2.0"):
+            return
         install_tree(
             self.prefix.share.heffte.testing, join_path(self.install_test_root, "testing")
         )
 
     def test_make_test(self):
         """build and run make(test)"""
+
+        if self.spec.satisfies("@:2.2.0"):
+            return
```

Comment:
The docs are probably still lacking on this feature.

I suggest raising the `SkipTest` exception so the test cab be flagged `SKIPPED` and picked up by CI/CDash.

```suggestion
            raise SkipTest("Test is not supported for versions @:2.2.0")
```